src/Robo/Plugin/Traits/DrupalVersionTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Usher\Robo\Plugin\Traits;
+
+/**
+ * Trait to determine what version of Drupal is being run.
+ */
+trait DrupalVersionTrait
+{
+    /**
+     * Check if version of Drupal in question is Drupal 7.
+     *
+     * @param string $drupalRootPath
+     *   The path to the Drupal root.
+     */
+    protected function drupalVersionIsD7(string $drupalRootPath): bool
```

Comment:
Could we do something like this?

- https://www.liquidweb.com/kb/version-drupal-running/

Seems slightly less brittle?

I don't want us to over think this though either if this works. 🤷🏼‍♂️